### PR TITLE
Dw 3419 concourse teams

### PR DESCRIPTION
**Pull request by ChrisScottThomas**
Creates teams in Concourse, maps users from Cognito Groups.

### CODE REVIEW

File: terraform/modules/concourse_web/web_config.tf [hunk 1]
```diff
@@ -100,6 +98,30 @@ locals {
       no_proxy                      = var.proxy.no_proxy
     }
   )
+
+  teams = templatefile(
+    "${path.module}/templates/teams.sh",
+    {
+      http_proxy         = var.proxy.http_proxy
```
Comment (mattburgess) [inline terraform/modules/concourse_web/web_config.tf:105]: All these proxy vars can disappear now :)

File: terraform/deploy/modules.tf [hunk 2]
```diff
@@ -177,7 +177,7 @@ module "cognito" {
   source = "../modules/cognito"
 
   clients = [
-    "concourse",
+    "dataworks"
```
Comment (dan-hill2802) [inline terraform/deploy/modules.tf:180]: better to keep the comma at the end

File: terraform/modules/concourse_web/web_config.tf (hunk 1 above)
Comment (ChrisScottThomas) [reply terraform/modules/concourse_web/web_config.tf]: Done.

File: terraform/deploy/modules.tf (hunk 2 above)
Comment (ChrisScottThomas) [reply terraform/deploy/modules.tf]: Done.

**Pull request merged**